README.md

```diff
@@ -99,7 +99,12 @@ var skPromise = Smooch.init({
         settingsInputPlaceholder: 'Your email address',
         settingsSaveButtonText: 'Save',
         settingsHeaderText: 'Email Settings',
-        settingsNotificationText: 'In case we\'re slow to respond you can <a href="#" data-ui-settings-link>leave us your email</a>.'
+        settingsNotificationText: 'In case we\'re slow to respond you can <a href data-ui-settings-link>leave us your email</a>.',
+        actionPaymentError: 'An error occurred while processing the card. <br> Please try again or use a different card.',
+        actionPaymentCompleted: 'Payment Completed',
+        messageError: 'An error occured while sending your message. Please try again.',
+        invalidFileError: 'Sorry, but only images are supported currently. Please choose a file with a supported extension (jpg, jpeg, png, gif, or bmp).',
+        messageIndicatorTitle: '{name} said...'
```

Comment:
Let's go with "(1) New message" & (2) "New messages" as name is will generate a 🤔  face most of the time for the user. Its very likely that its the first time you see this name